gulp/isotest.js

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright 2015 Palantir Technologies, Inc. All rights reserved.
+ */
+"use strict";
+
+module.exports = (gulp, plugins, blueprint) => {
+    const rs = require("run-sequence").use(gulp);
+    const mocha = require("gulp-mocha");
+
+    blueprint.projectsWithBlock("isotest").forEach((project) => {
+        gulp.task(`isotest-${project.id}`, [`typescript-compile-${project.id}`], (done) => {
```

Comment:
look at how the `typescript-compile` tasks are defined in typescript.js. you'll want to do something similar here. depending on `["typescript-compile-*"]` will auto-expand the * to corresponding project id.